src/libraries/System.Private.CoreLib/src/System/Text/Utf8Span.cs

```diff
@@ -225,7 +241,20 @@ public override string ToString()
             // TODO_UTF8STRING: Since we know the underlying data is immutable, well-formed UTF-8,
             // we can perform transcoding using an optimized code path that skips all safety checks.
 
+#if CORECLR || NETCOREAPP
             return Encoding.UTF8.GetString(Bytes);
+#else
+            byte[] buffer = ArrayPool<byte>.Shared.Rent(Length);
+            try
+            {
+                Bytes.CopyTo(buffer);
+                return Encoding.UTF8.GetString(buffer, 0, Length);
+            }
+            finally
+            {
+                ArrayPool<byte>.Shared.Return(buffer);
+            }
```

Comment:
Doesn't Encoding.GetString have a pointer-based overload in netstandard? Assuming yes, it'd be better to use that rather than renting a buffer and copying.